src/test/librbd/fsx.cc

```diff
@@ -2504,8 +2504,10 @@ __aio_rw(int rw, int fd, char *buf, unsigned len, unsigned offset)
 	} while (ret == -EINTR);
 
 	if (ret != 1) {
-		if (ret == 0)
+		if (ret == 0) {
```

Comment:
@johnzzpcrystal Sounds good to me -- just force push your changes to the existing branch.